doc/engine/What's-new.md

```diff
@@ -1,190 +1,252 @@
+can be exported to a zip file containing a set of output files. For
+instance an event based risk calculation with two realizations and
+four loss types in the past could print something like the following:
+
+```
+  id | output_type | name
+ 515 | Aggregate Loss Curve | aggregate loss curves. loss_type=contents hazard=430||gmf||GMF rlz-332
+ 514 | Aggregate Loss Curve | aggregate loss curves. loss_type=contents hazard=431||gmf||GMF rlz-333
+ 519 | Aggregate Loss Curve | aggregate loss curves. loss_type=fatalities hazard=430||gmf||GMF rlz-332
+ 518 | Aggregate Loss Curve | aggregate loss curves. loss_type=fatalities hazard=431||gmf||GMF rlz-333
+ 512 | Aggregate Loss Curve | aggregate loss curves. loss_type=nonstructural hazard=430||gmf||GMF rlz-332
+ 513 | Aggregate Loss Curve | aggregate loss curves. loss_type=nonstructural hazard=431||gmf||GMF rlz-333
+ 516 | Aggregate Loss Curve | aggregate loss curves. loss_type=structural hazard=430||gmf||GMF rlz-332
+ 517 | Aggregate Loss Curve | aggregate loss curves. loss_type=structural hazard=431||gmf||GMF rlz-333
+```
+
+Now it will print only one line:
+
+```
+  id | output_type | name
+ 419 | datastore | agg_curve-rlzs
+```
+
+When exporting the composite output, 8 XML files will be generated, in
+the same format as before, plus a zip file containing all of them.
+When using the Web UI the zip file will be available for download.
 
 Support for different platforms
 ----------------------------------------------------
 
-OpenQuake 1.5 fully supports both Ubuntu 12.04 and Ubuntu 14.04
-and we provide packages for both platforms. However,
-starting from OpenQuake 1.6 *we will release packages only for Ubuntu 14.04*.
-Ubuntu 12.04 will still be supported but you will have to install manually some
-dependencies which are not in the repositories of Ubuntu 12.04. The reason
-for the change is that the HDF5 libraries for Ubuntu 12.04 are
-too old (over 4 year old), buggy and less efficient compared to 
-the ones for Ubuntu 14.04, which is now our official development platform.
-It is too expensive for us to mantain compatibility with such ancient
-software, so users wishing to use OpenQuake 1.6 on Ubuntu 12.04
-will have to install manually the library h5py (version 2.2.1)
-and its dependencies. We will provide instructions for that in
-the next release, since for the moment this is not necessary.
-
-We have detailed instructions for installing the engine on CentOS 7
-and Fedora and in general on [Red Hat Enterprise Linux clones]
+OpenQuake 1.6 supports Ubuntu from release 12.04 up to 15.10. 
+We provide official packages for the long term releases 12.04 and 14.04.
+We were able to extend the support to
+Ubuntu 12.04 by backporting the package `python-h5py` from Ubuntu 14.04.
+So *Ubuntu 12.04 is still supported, even if it is deprecated*.
+
+We have official packages also for CentOS 7
+and Fedora and in general for [Red Hat Enterprise Linux clones].
```

Comment:
No official packages for Fedora. Our CentOS package will work on Fedora 19-22 but not on Fedora 22 (Django 1.8 issues)